### PR TITLE
Nested flying carpet hack fix

### PR DESCRIPTION
**Pull request by jridgewell**
There’s a clever hack where you can put a flying-carpet inside a
fixed-position element, make it 100% size, and you have a full-page ad
interstitial. Good thing flying-carpets are experimental.

### CODE REVIEW

File: src/ad-helper.js [hunk 1]
```diff
@@ -18,51 +18,65 @@
...
+    if (CONTAINERS[el.tagName]) {
+      // The containers must not themselves be contained in a fixed-position
+      // element. Continue the search.
+      containers++;
+      hasFixedAncestor = false;
+    } else if (isPositionFixed(el, win)) {
       // Because certain blessed elements may contain a position fixed
       // container (which contain an ad), we continue to search the
       // ancestry tree.
       hasFixedAncestor = true;
```
Comment (erwinmombay) [inline src/ad-helper.js]: can we break early here if we've found a fixed ancestor?

Comment (jridgewell) [inline src/ad-helper.js]: No, because it may be contained by a `flying-carpet` which allows it.

**Comment by erwinmombay** [comment]
LGTM, deferring to @dvoytenko for final LGTM

**Comment by dvoytenko** [comment]
LGTM

**Pull request merged**